#!/usr/bin/env python3

"""
    Assetto Corsa Target Params
    Jinsun Park
    (zzangjinsun@3secondz.com)
"""

PACKAGE = "assetto_corsa"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("vxInput", double_t, 0, "vxInput", 30.0, 0.0, 200.0)
gen.add("accMax", double_t, 0, "accMax", 1.0, 0.0, 10.0)
gen.add("axMax", double_t, 0, "axMax", 1.0, 0.0, 10.0)
gen.add("ayMax", double_t, 0, "ayMax", 1.0, 0.0, 10.0)

exit(gen.generate(PACKAGE, "ac_target", "ACTargetParams"))
